internal/httputil/server.go

```diff
@@ -1,50 +1,42 @@
 package httputil // import "github.com/pomerium/pomerium/internal/httputil"
 
 import (
+	"context"
 	"crypto/tls"
 	"fmt"
 	stdlog "log"
 	"net"
 	"net/http"
-	"strings"
+	"os"
+	"os/signal"
+	"sync"
+	"syscall"
+	"time"
 
-	"github.com/pomerium/pomerium/internal/cryptutil"
 	"github.com/pomerium/pomerium/internal/log"
+	"github.com/pomerium/pomerium/internal/urlutil"
 )
 
-// NewTLSServer creates a new TLS server given a set of options, handlers, and
+// NewServer creates a new HTTP(s) server given a set of options, handlers, and
 // optionally a set of gRPC endpoints as well.
```

Comment:
Is this still accurate with this call signature?

Reply:
Good catch.